x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -111,6 +111,10 @@ export class HeadlessChromiumDriverFactory {
     private binaryPath: string,
     private basePath: string
   ) {
+    const dataDir = getDataPath();
+    fs.mkdirSync(dataDir, { recursive: true });
+    this.userDataDir = fs.mkdtempSync(path.join(dataDir, 'chromium-'));
```

Comment:
Turns out that on the Jest workers there's nothing that creates the `data` dir except for `../src/setup_node_env` when it initializes the node register cache, which I removed from `scripts/update_vscode_config` and broke this test.